DKImagePickerController/DKAssetGroupDetailVC.swift

```diff
@@ -267,7 +267,7 @@ internal class DKAssetGroupDetailVC: UICollectionViewController {
     private lazy var selectGroupButton: UIButton = {
         let button = UIButton()
         button.addTarget(self, action: "showGroupSelector", forControlEvents: .TouchUpInside)
-        button.setTitleColor(UIColor.blackColor(), forState: .Normal)
+        button.setTitleColor(UIColor.whiteColor(), forState: .Normal)
```

Comment:
This is a customization only for me. This shouldn't get included to this PR. Sorry